src/Build.OM.UnitTests/Construction/SolutionFile_Tests.cs

```diff
@@ -16,6 +16,84 @@ namespace Microsoft.Build.UnitTests.Construction
     /// </summary>
     public class SolutionFile_Tests
     {
+        [Theory]
+        [InlineData(@"
+                {
+                  ""solution"": {
+                    ""path"": ""C:\\notAPath\\MSBuild.Dev.sln"",
```

Comment:
Are the paths in `.slnf` files really absolute? That feels pretty broken to me.

Reply:
That's the default. Not only are they absolute, you might notice the @ symbol at the front—they're escaped.